server_addon/maya/client/ayon_maya/plugins/publish/extract_rig.py

```diff
@@ -2,17 +2,15 @@
 """Extract rig as Maya Scene."""
 import os
 
+from ayon_maya.api.lib import maintained_selection
+from ayon_maya.api.plugin import MayaExtractorPlugin
```

Comment:
Doesn't directly importing a `pyblish plug-in` into this space mean that Pyblish logic will also pick up that class and thus basically will also load that `MayaExtractorPlugin` base class as a plug-in? Isn't that right @iLLiCiTiT ?

Reply:
You are right.

Reply:
In that case @antirotor all these imports need to change so that we load the module instead and inherit from the plug-in that way. Similar to how the code looked before.

Reply:
yeah, you're right. I forgot about this quirk. Hopefuly easy job for find&replace.